### ISSUE TITLE
Configurable sign up link in navbar

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/chaise#542]
We have a configurable Login link in the Chaise navbar but we do not have a Sign up link.

**Comment by howdyjessie** [comment]
@hongsudt This might not be as simple as previously thought. We can't just make a static Sign Up link that goes to globus.org because what if a deployment doesn't use Globus, like BDDS? How do non-Globus deployments make accounts?

**Comment by karlcz** [comment]
I thought the described approach was a chaise-config entry to provide a sign-up link to use in the navbar. If no link is configured, there should be no sign-up button at all.

**Comment by howdyjessie** [comment]
@karlcz I must've missed/forgot that discussion. Okay well if the navbar is just using whatever's set in chaise-config.js, then this is a simple issue again. :)

**Comment by robes** [comment]
@karlcz @howdyjessie The caveat here is that the link should change depending on the user's log in state:

- not logged in =\> "Sign up"
- logged in =\> "\<username\>"

Those might link to the same href, in Facebase I think we will always link to the accounts.facebase.org site. Other applications might have different requirements.

@crisaless or @svoinea Might be able to point you to the code that is already in use for Facebase. I assume some other projects also use that code.

**Comment by crisaless** [comment]
The old code is live on the production server but is not in master anymore - but here's a 'reconstruction':

The html for the user links were being generated by files on facebase-www - and i think we're trying to get away from that. But on the search page, it was coming in from `facebase-www/www/_includes/body_nav_data.html` (which is pulled into the `www/data/index.html.in` template)

```
<ul class="nav navbar-nav navbar-right"> 
	<li class="login"><a id="fblogin_link" href="javascript:void(0)" onclick="loginURL();">Log in</a></li>
	<li class="signup"><a id="fbsignup_link" href="https://accounts.facebase.org/SignIn#step=signup">Sign up</a></li>
	<li class="user"><a id="fblogin_user" href="https://accounts.facebase.org"></a></li>
	<li class="logout"><a id="fblogout_user" href="javascript:void(0)" onclick="logoutURL();">Logout</a></li>
</ul>
```
then `facebase-www/www/_includes/header_data.html` includes a link to `/js/facebase-userlinks.js` includes the js to show/hide stuff:

https://github.com/informatics-isi-edu/facebase-www/blob/397f03facdc28fd883f37718aa0de75e3a65bb42/www/js/facebase-userlinks.js

**Comment by howdyjessie** [comment]
This issue has been addressed in PR #614.
